packages/grid/x-data-grid-pro/src/components/DataGridProVirtualScroller.tsx

```diff
@@ -101,7 +101,10 @@ const VirtualScrollerDetailPanels = styled('div', {
   name: 'MuiDataGrid',
   slot: 'DetailPanels',
   overridesResolver: (props, styles) => styles.detailPanels,
-})({});
+})({
+  position: 'relative',
+  zIndex: -1, // To make scroll grabbing work in Safari (see https://github.com/mui/mui-x/issues/4360)
```

Comment:
It broke https://deploy-preview-4610--material-ui-x.netlify.app/x/react-data-grid/master-detail/#disable-detail-panel-content-scroll

The detail panel should go over the pinned columns.

<img width="250" src="https://user-images.githubusercontent.com/42154031/164745098-4abe5480-96f6-434a-833a-a70668cb4987.png" alt="image" style="max-width: 100%;">

Reply:
Yes, without `zIndex: -1` opening detail panel causes this issue https://github.com/mui/mui-x/issues/4360
I'll try to find another solution, this scroll issue keeps haunting me 😀

Reply:
I don't see a simple solution to that, so I propose to remove `zIndex: -1` and ignore scroll grabbing issue, because:
- it's Safari only
- it appears only when detail panel is open
- IMHO not a lot of people grab the scrollbar on MacOS, if otherwise - let's wait for user reports